osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -121,7 +121,11 @@ private void load()
                 }
             });
 
-            PositionBindable.BindValueChanged(_ => Position = HitObject.StackedPosition);
+            PositionBindable.BindValueChanged(_ =>
+            {
+                Position = HitObject.StackedPosition;
+                childAnchorPosition = null;
+            });
```

Comment:
The fact that this fix had to be applied in post worries me.

Given that the `childAnchorPosition` thing is entirely dependent on `OriginPosition` and `DrawSize`, I would prefer that rather than having a manual invalidation flow, a framework `LayoutValue` with `Invalidation.DrawSize | Invalidation.MiscGeometry` was used, if it doesn't destroy the optimisation. It would make me much more comfortable as that alternative fix would be obviously and unquestionably correct under any circumstance.

Reply:
Seems to work well (https://github.com/peppy/osu/commit/5970a68e2d12f4f8fc3a118cf68da7a69c1f1fcf).